.eslintrc

```diff
@@ -12,6 +12,7 @@
         "indent": 2,
         "prefer-const": 1,
         "react/jsx-curly-spacing": [ 2, "always" ],
-        "react/jsx-filename-extension": 0
+        "react/jsx-filename-extension": 0,
+        "semi": "off"
```

Comment:
Why we remove semicolon (;) ?
Though JS has automatic semicolon insertion, it is well known to make it more readable and automatic insertion sometimes make miss interpretion.

See: http://inimino.org/~inimino/blog/javascript_semicolons

Reply:
Our team want to experiment with this project because we never try to develop no-semicolon javascript project.